lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -335,21 +335,27 @@ struct _MaterialXData
 {
     _MaterialXData()
     {
-        _mtlxSearchPath = HdMtlxSearchPaths();
+        try {
+            _mtlxSearchPath = HdMtlxSearchPaths();
```

Comment:
Any error in this code path will throw a C++ exception that will completely break the Maya viewport. Making sure we catch any issue here.